exceptions/CustomExceptions.java

```diff
@@ -0,0 +1,29 @@
+package exceptions;
+
+public class CustomExceptions {
+
+    //Custom exception
+    public static class DuplicatedDependencyFoundException extends Exception{
+        public DuplicatedDependencyFoundException(){
+            super("Already exists");
```

Comment:
Also please be more specific in the error messaging, for example here... what already exists?

Check other exceptions as well

Reply:
used lambda to pass the value of s to the exception.